### PR TITLE
[new release] rawlink (1.2)

### PR DESCRIPTION
**Pull request by haesbaert**
Portable library to read and write raw packets

- Project page: <a href="https://github.com/haesbaert/rawlink">https://github.com/haesbaert/rawlink</a>
- Documentation: <a href="https://haesbaert.github.io/rawlink/">https://haesbaert.github.io/rawlink/</a>

##### CHANGES:

* Fix linking against non-lwt rawlink
* Implement promiscuous mode
* Update caml_ calls in rawlink_stubs.c

### CODE REVIEW

File: packages/rawlink/rawlink.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+license: "ISC"
+homepage: "https://github.com/haesbaert/rawlink"
+bug-reports: "https://github.com/haesbaert/rawlink/issues"
+dev-repo: "git+https://github.com/haesbaert/rawlink.git"
+doc: "https://haesbaert.github.io/rawlink/"
+build: [
+  [ "dune" "subst" ] {pinned}
```
Comment (mseri) [inline packages/rawlink/rawlink.1.2/opam:10]:
```suggestion
  [ "dune" "subst" ] {dev}
```

File: packages/rawlink/rawlink.1.2/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+homepage: "https://github.com/haesbaert/rawlink"
+bug-reports: "https://github.com/haesbaert/rawlink/issues"
+dev-repo: "git+https://github.com/haesbaert/rawlink.git"
+doc: "https://haesbaert.github.io/rawlink/"
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ] ]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
```
Comment (mseri) [inline packages/rawlink/rawlink.1.2/opam:14]:
```suggestion
  "dune"
```
The build flag should not be used with dune for forward compatibility reasons

Comment (haesbaert) [reply packages/rawlink/rawlink.1.2/opam]: Thanks

**Comment by mseri** [comment]
Thanks

**Pull request merged**